app/views/admin/tracks/index.html.haml

```diff
@@ -7,75 +7,76 @@
         Categorize events in your conference
 .row
   .col-md-12
-    %table.table.table-hover.table-striped.table-bordered.datatable#tracks
-      %thead
-        %th ID
-        %th Name
-        %th Description
-        %th Room
-        %th Start Date
-        %th End Date
-        %th Submitter
-        %th Included in Cfp
-        %th State
-        %th Actions
-      %tbody
-        - @tracks.each do |track|
-          %tr
-            %td
-              = track.id
-            %td{ style: 'padding: 15px 0px 0px 10px;' }
-              = link_to admin_conference_program_track_path(@conference.short_title, track), class: 'btn' do
-                %span.label{style: "background-color: #{track.color}; color: #{ contrast_color(track.color) }"}
-                  = track.name
-            %td
-              %p
-                = markdown(truncate(track.description))
-            %td
-              = track.room.try(:name)
-            %td
-              = track.start_date.strftime('%A, %B %-d. %Y') if track.start_date
-            %td
-              = track.end_date.strftime('%A, %B %-d. %Y') if track.end_date
-            %td
-              = link_to track.submitter.name, admin_user_path(track.submitter) if track.self_organized?
-            %td.text-center{ 'id' => "cfp_switch_#{track.id}", 'data-order' => track.cfp_active.to_s }
-              = check_box_tag "#{@conference.short_title}_#{track.short_name}", track.id, track.cfp_active,
-                class: 'switch-checkbox', method: :patch,
-                url: toggle_cfp_inclusion_admin_conference_program_track_path(@conference.short_title, id: track.short_name)+"?included=",
-                data: { size: 'small',
-                        on_color: 'success',
-                        off_color: 'warning',
-                        on_text: 'Yes',
-                        off_text: 'No' }
+    - if !@tracks.empty?
```

Comment:
For booths you did, if booths.any?, why do you prefer this other way here for the tracks?

Reply:
i saw @booths.any? is used at some other place too in the booths' index.html.haml , so i used any and on tracks i used empty as i always use it because sometimes it is faster than any.
https://mensfeld.pl/2015/06/empty-vs-blank-vs-any-why-you-should-not-use-any-to-test-if-theres-anything-in-the-array/
i read it from here.